docs/data/material/getting-started/learn/learn.md

```diff
@@ -1,61 +1,65 @@
-# Learn MUI
+# Learning Resources
 
-<p class="description">New to MUI? It's easy to learn if you know where to start! This guide will help you to get started quickly.</p>
+<p class="description">New to Material UI? Get up to speed quickly with our curated list of learning resources.</p>
 
-Developers come to MUI from different backgrounds and with different learning styles. Whether you prefer a more theoretical or practical approach, we hope you'll find this section helpful.
-Like any unfamiliar technology, MUI does have a learning curve. With practice and some patience, you will soon get the hang of it.
+## Your first component
 
-## First example
+The following code snippet demonstrates a basic Material UI app that features a `<Button>` component:
 
-The [usage page](/getting-started/usage/#quick-start) contains a small MUI example with a live editor.
-Even if you don't know anything about MUI yet, try changing the code and see how it affects the result.
+```jsx
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '@mui/material/Button';
+
+function App() {
+  return (
+    <Button variant="contained" color="primary">
+      Hello World
+    </Button>
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+In the interactive demo below, try changing the code and see how it affects the output. (Hint: change `variant` to `"outlined"` and `color` to `"secondary"`. For more options, see the [`Button` component page](components/buttons/).)
```

Comment:
```suggestion
In the interactive demo below, try changing the code and see how it affects the output. (Hint: change `variant` to `"outlined"` and `color` to `"secondary"`. For more options, see the [`Button` component page](/components/buttons/).
```

Reply:
The last period there should be inside of the parentheses, since it's a complete sentence that stands on its own (as opposed to an additional clause, like this example 😁).

Reply:
Let's trust Sam he is native :D. It is the same in my language so it makes sense :)